N new.csv

```diff
@@ -0,0 +1,36 @@
+Lemma,Sense,Word Class,Year,Quotation,Source,Note
+奶糖,/,noun,1886,,洪士提反譯『萬國藥方』卷一,
+難產,/,noun,1822,LABOUR（Difficult labour，拆副難產；難產. There is a volume on difficult labours, called 達生編 馬禮遜 英華字典,
```

Comment:
Add proper punctuation marks, eg book title.

Quite a number of other entries are missing these punctuation marks. Fix them.